lib/inquisition/rails_best_practices/vulnerability.rb

```diff
@@ -0,0 +1,86 @@
+          'RailsBestPractices::Reviews::NeedlessDeepNestingReview',
+          'RailsBestPractices::Reviews::ReplaceComplexCreationWithFactoryMethodReview',
+          'RailsBestPractices::Reviews::UseModelAssociationReview',
+          'RailsBestPractices::Reviews::UseMultipartAlternativeAsContentTypeOfEmailReview',
+          'RailsBestPractices::Reviews::UseObserverReview',
+          'RailsBestPractices::Reviews::UseScopeAccessReview'
+        ],
+        style: [
+          'RailsBestPractices::Lexicals::LongLineCheck',
+          'RailsBestPractices::Lexicals::RemoveTabCheck',
+          'RailsBestPractices::Lexicals::RemoveTrailingWhitespaceCheck',
+          'RailsBestPractices::Reviews::HashSyntaxReview',
+          'RailsBestPractices::Reviews::NotUseTimeAgoInWordsReview',
+          'RailsBestPractices::Reviews::OveruseRouteCustomizationsReview',
+          'RailsBestPractices::Reviews::ReplaceInstanceVariableWithLocalVariableReview',
+          'RailsBestPractices::Reviews::RestrictAutoGeneratedRoutesReview',
+          'RailsBestPractices::Reviews::SimplifyRenderInControllersReview',
+          'RailsBestPractices::Reviews::UseParenthesesInMethodDefReview',
+          'RailsBestPractices::Reviews::UseQueryAttributeReview',
+          'RailsBestPractices::Reviews::UseSayWithTimeInMigrationsReview',
+          'RailsBestPractices::Reviews::SimplifyRenderInViewsReview'
+        ],
+        bug_risk: [
+          'RailsBestPractices::Reviews::CheckSaveReturnValueReview',
+          'RailsBestPractices::Reviews::DefaultScopeIsEvilReview',
+          'RailsBestPractices::Reviews::NotRescueExceptionReview',
+          'RailsBestPractices::Reviews::CheckDestroyReturnValueReview'
+        ],
+        unused_code: [
+          'RailsBestPractices::Reviews::RemoveUnusedMethodsInControllersReview',
+          'RailsBestPractices::Reviews::RemoveEmptyHelpersReview',
+          'RailsBestPractices::Reviews::RemoveUnusedMethodsInHelpersReview',
+          'RailsBestPractices::Reviews::RemoveUnusedMethodsInModelsReview'
+        ],
+        performance: [
+          'RailsBestPractices::Reviews::AlwaysAddDbIndexReview',
+          'RailsBestPractices::Reviews::UseTurboSprocketsRails3Review'
+        ],
+        security: [
+          'RailsBestPractices::Reviews::ProtectMassAssignmentReview',
+          'RailsBestPractices::Reviews::NotUseDefaultRouteReview'
+        ],
+        duplication: ['RailsBestPractices::Reviews::UseBeforeFilterReview']
+      }.freeze
+
+      def initialize(error)
+        @error = error
+      end
+
+      def to_h
+        {
+          category: find_category(error),
+          severity: :low,
+          message: error.message,
+          path: error.short_filename,
+          line: error.line_number.to_i
+        }
+      end
+
+      private
+
+      attr_reader :error
+
+      def find_category(error)
+        ERROR_TYPES.each do |category, error_types|
```

Comment:
Can we use [detect](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-detect) here?